src/views/v2/month/top-bar/datepicker.php

```diff
@@ -41,7 +41,7 @@ class="tribe-events-c-top-bar__datepicker-input tribe-common-a11y-visual-hide"
 		data-js="tribe-events-top-bar-date"
 		id="tribe-events-top-bar-date"
 		name="tribe-events-views[tribe-bar-search]"
-		value="<?php echo esc_attr( tribe_events_template_var( [ 'bar', 'date' ], '' ) ); ?>"
+		value="<?php echo esc_attr( $the_date->format('Y-m-d') ); ?>"
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening bracket; 0 found
* :no_entry_sign: Expected 1 spaces before closing bracket; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: b768fdeaef25eb321b19a3976464ce6c</sub>